ofrak_patch_maker/ofrak_patch_maker/toolchain/gnu_vbcc_m68k.py

```diff
@@ -61,16 +61,11 @@ def __init__(
                 "--no-dynamic-linker",
             )
 
-        vbcc_compiler_optimization_map = {
-            CompilerOptimizationLevel.NONE: "",
-            CompilerOptimizationLevel.SOME: "",
-            CompilerOptimizationLevel.SPACE: "-size",
-            # TODO: Look into O=16384 for full cross-module optimizations. Bit encoded.
-            CompilerOptimizationLevel.FULL: "-speed",
-        }
-        self._compiler_flags.append(
-            vbcc_compiler_optimization_map[self._config.compiler_optimization_level]
-        )
+        # TODO: Look into O=16384 for full cross-module optimizations. Bit encoded.
+        if self._config.compiler_optimization_level is CompilerOptimizationLevel.SPACE:
+            self._compiler_flags.append("-size")
+        elif self._config.compiler_optimization_level is CompilerOptimizationLevel.FULL:
```

Comment:
I think it's the more Pythonic way and it reads better (`"if configured optimization level is FULL optimization"`). In this case we are literally checking if two object refs are the same because of how Python enums work, although that's not the high-level intention.